buildscripts/libs/build_cmake.sh

```diff
@@ -29,7 +29,7 @@ fi
 software=$name-$version
 cd ${JEDI_STACK_ROOT}/${PKGDIR:-"pkg"}
 url="https://cmake.org/files/v${version%.*}/$software.tar.gz"
-[[ -d $software ]] || ( $WGET $url; tar -xf $software.tar.gz )
```

Comment:
If the download gets interrupted, a corrupted .tar.gz file remains in pkg. wget will not overwrite it but instead download it again as a .tar.gz.1 file.